ups/meas_mosaic.table

```diff
@@ -1,6 +1,8 @@
 setupRequired(afw)
 setupRequired(numpy)
+setupRequired(ndarray)
```

Comment:
This table file is missing python, boost, gsl, eigen and cfitsio if you compare the list to the `.cfg` file.